relay/sources/Makefile

```diff
@@ -47,19 +47,26 @@ relay-api/virtualenv/virtualenv.py: relay-api/virtualenv.tgz
 rudder-pkg/rudder_plugins_key.pub:
 	$(GET) rudder-pkg/rudder_plugins_key.pub https://repository.rudder.io/apt/rudder_apt_key.pub $(RUDDER_PLUGINS_KEY_SHA256)
 
-man-source: target/man-source/rudder-relayd.1;
+man-source: target/man-source/rudder-relayd.1 autocomplete-source;
 
 target/man-source/%: man/%.adoc
 	mkdir -p target/man-source
 	asciidoctor -D target/man-source -b manpage $<
 
-man: target/man/rudder-relayd.1.gz;
+autocomplete-source:
+	PYTHONPATH=$(CURDIR)/rudder-pkg/lib/rudder-pkg:$$PYTHONPATH; cd man && docopt-completion $(CURDIR)/rudder-pkg/rudder-pkg --manual-bash && cd -
```

Comment:
would it be possible to generate this file in the target directory instead? (and created file path as target)

Reply:
fixed